webchanges/reporters.py

```diff
@@ -621,24 +621,37 @@ class StdoutReporter(TextReporter):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self._has_color = sys.stdout.isatty() and self.config.get('color', False)
+        color_map = {
+            'default': 39,
+            'black': 30,
+            'red': 31,
+            'green': 32,
+            'yellow': 33,
+            'blue': 34,
+            'magenta': 35,
+            'cyan': 36,
+            'lightgray': 37,
+            'darkgray': 90,
```

Comment:
Correct name is 'lightblack' (also see https://github.com/tartley/colorama/blob/master/colorama/ansi.py)